docs/service_architecture.md

```diff
@@ -27,7 +27,7 @@ but here is a brief recap of the TUF roles and corresponding key hierarchy:
   which lists the IDs of the root, targets, snapshot, and timestamp public keys.
   Clients use these public keys to verify the signatures on all the metadata files
   in the repository. This key is held by a collection owner, and should be kept offline
-  and safe, more so than any other key.
+  and safe, more so than any other keys.
```

Comment:
Did some checking - it should indeed be singular, if you can change it back to `key` that would be great :)